plover/engine.py

```diff
@@ -341,9 +341,18 @@ def _consume_engine_command(self, command):
         elif command == 'LOOKUP':
             self._trigger_hook('lookup')
         else:
+            # The remaining commands might require an argument after a colon
             command_args = command.split(':', 1)
-            command_fn = registry.get_plugin('command', command_args[0]).obj
-            command_fn(self, command_args[1] if len(command_args) == 2 else '')
+            if command_args[0] == 'SET_CONFIG':
+                key, value = tuple(command_args[1].split(','))
+                oldvalue = self[key]
+                try:
+                    self[key] = (type(oldvalue))(value)
+                except TypeError:
+                    log.error("Could not cast config value string to required type.")
+            else:
+                command_fn = registry.get_plugin('command', command_args[0]).obj
+                command_fn(self, command_args[1] if len(command_args) == 2 else '')
```

Comment:
The new command should behave as a plugin, even if it is builtin one. So this whole hunk should disappear.

Reply:
I considered doing this like the built-in macros. I guess I was hesitant because there aren't any built-in PLOVER: commands that currently operate like that and doing so would entail making a new folder, file, and entry point. I'll make the change and push.